src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -40,6 +53,14 @@
  */
 public final class SiteUtil {
 
+    /** Class name and their corresponding parent module name.. */
+    private static final Map<Class<?>, String> CLASS_TO_PARENT_MODULE = Map.ofEntries(
```

Comment:
Just curious, why Map#ofEntries instead of Map#of?

Reply:
I find it a bit more readable than `Map#of`. I was looking at https://stackoverflow.com/a/6802502/9553927 and went with `Map#ofEntries`. `Map#of` would have worked fine since we only have 8 elements.